Plugins/Triage/Retrieve-Items.ps1

```diff
@@ -163,14 +202,23 @@ process{
 
                 foreach ($DirItem in $DirItems){
```

Comment:
Is `$DirItems` ever set? I'm not seeing a `$DirItems` variable before this foreach loop

Reply:
Yep, line 203 right above that foreach.